### ISSUE TITLE
main landmark not being picked up

### ISSUE THREAD

**Issue by stevefaulkner** [matatk/landmarks#8]
have tolook for examples

**Comment by matatk** [comment]
Is this really not being picked up (semantically), or is it a visual problem?

If it is a visual problem, that the border is not being shown (I've tried to make it use CSS `border`, rather than `outline`, as that might help, but sometimes it's not possible to highlight the area via its border).

Idea: could we use any CSS transforms to e.g. apply a temporary overall filter highlight to the *entire region*?  If we did, how would this affect the "permanent" setting in the preferences?  Should we allow the colour of the highlight to be configurable?

Unfortunately sometimes the highlights simply don't match the area on the page, because of the design on the page, so the highlight highlights something completely different, visually, than where the region appears to be.
